bot/resources/tags/repl.md

```diff
@@ -2,14 +2,19 @@
 embed:
     title: "Read-Eval-Print Loop (REPL)"
 ---
-A REPL is an interactive language shell environment. It first **reads** one or more expressions entered by the user, **evaluates** it, yields the result, and **prints** it out to the user. It will then **loop** back to the **read** step.
+A REPL is an interactive shell where you can execute individual lines of code one at a time, like so:
+```python-repl
+>>> x = 5
+>>> x + 2
+7
+>>> for i in range(3):
+...     print(i)
+...
+0
+1
+2
+>>>
+```
+To enter the REPL, run `python` in the command line without any arguments. The `>>>` or `...` at the start of some lines are prompts to enter code, and indicate that you are in the Python REPL. Any other lines show the output of the code.
```

Comment:
```suggestion
To enter the REPL, run `python` (`py` on Windows) in the command line without any arguments. The `>>>` or `...` at the start of some lines are prompts to enter code, and indicate that you are in the Python REPL. Any other lines show the output of the code.
```
Suggested by MoyenChat in [#community-meta](https://discord.com/channels/267624335836053506/429409067623251969/1146794080861425725)